### PR TITLE
Add veracrypt.rb

### PR DESCRIPTION
**Pull request by soredake**

### CODE REVIEW

File: Casks/veracrypt.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask :v1 => 'veracrypt' do
+  version '1.0f-2'
+  sha256 '91a28808b16a2a6f664b2e22d4f511416154d293ac3dcc576605a4559aabc93f'
+
+  url 'https://a.pomf.cat/fbwbrm.dmg'
```
Comment (fanquake) [inline Casks/veracrypt.rb]: Where does this URL come from?

**Comment by fanquake** [comment]
Should we be making a note of the OSXFUSE requirement?

There is also a PGP signature available, we should be making use of that.

File: Casks/veracrypt.rb (hunk 1 above)
Comment (fenque) [inline Casks/veracrypt.rb]: It was necessary to use this file sharing, trying to gain a permanent link with codeplex impossible.

**Comment by fenque** [comment]
Ok, I will do it.

File: Casks/veracrypt.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/veracrypt.rb]: We can’t accept this, then. Links must be trustworthy.

**Pull request closed without merging**